test/browser_tests/shared_objects/multiselect.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+var _multiSelect =
+  element( by.css( '.cf-multi-select' ) );
```

Comment:
Maybe turn this into `elements.all( by.css( '.cf-multi-select' ) ).first();` (I think that's right).. so we don't get a bunch `WARNING - more than one element found for locator By(css selector, .cf-multi-select) - the first result will be used`